packages/components/src/button/index.tsx

```diff
@@ -26,7 +26,7 @@ type AnchorProps = OwnProps &
 		href: NonUndefined< AnchorElementProps[ 'href' ] >;
 	};
 
-type ButtonProps = OwnProps & Omit< ButtonHTMLAttributes< HTMLButtonElement >, 'href' >;
+export type ButtonProps = OwnProps & Omit< ButtonHTMLAttributes< HTMLButtonElement >, 'href' >;
```

Comment:
I'm not exactly sure why there's an explicit need to `Omit` the `href` attributes from `ButtonHTMLAttributes< HTMLButtonElement >` ?

Reply:
This component renders and `a` if there's an `href`, or a `button` otherwise. So the passthrough props are different depending on the presence of the `href`. We won't render a `button` element if an `href` is passed.